### PR TITLE
[Home] Add tracking

### PR DESCRIPTION
**Pull request by damassi**
Addresses https://artsyproduct.atlassian.net/browse/GRO-526 

> ⚠️  Depends on https://github.com/artsy/cohesion/pull/242

Adds tracking to home.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
   },
   "dependencies": {
     "@artsy/backbone-mixins": "2.0.0",
-    "@artsy/cohesion": "2.22.0",
+    "@artsy/cohesion": "4.4.0",
```
Comment (damassi) [inline package.json:80]: @dzucconi - did you ever learn why force hadn't been updated with the latest major versions?

Comment (dzucconi) [reply package.json:80]: https://github.com/artsy/force/pull/8454 was failing due to some unrelated CI thing (something about the image) — it's green now so I just merged.

Comment (dzucconi) [reply package.json:80]: I'm pretty sure major versions don't automatically merge though.

Comment (damassi) [reply package.json:80]: oh cool, yeah didn't know if there was some helpers RFC work in progress blocking that from being merged 👍

File: src/v2/Apps/Home/Components/HomeFeaturedGalleriesRail.tsx [hunk 2]
```diff
@@ -150,9 +191,11 @@ export const HomeFeaturedGalleriesRailFragmentContainer = createFragmentContaine
               ... on Profile {
                 ...FollowProfileButton_profile
                 internalID
+                isFollowed
                 name
                 slug
                 href
+                # following
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeFeaturedGalleriesRail.tsx:198]: ?

**Review by dzucconi — APPROVED** [review]
😅 looks good. Feel free to merge when Cohesion is ready

File: src/v2/Apps/Home/Components/HomeFeaturedGalleriesRail.tsx (hunk 2 above)
Comment (damassi) [reply src/v2/Apps/Home/Components/HomeFeaturedGalleriesRail.tsx:198]: ah, this was me guessing at a field -- it's actually `isFollowed`

**Pull request merged**